test_requirements.txt

```diff
@@ -9,6 +9,7 @@ pytest-capturelog
 pytest-watch
 pytest-xdist
 mock
+ipdb
```

Comment:
No, this can't be used in production. It's optionally used during development in place of pdb, with the advantage of tab-completion.